### PR TITLE
Modify valid? method

### PR DESCRIPTION
**Pull request by hungnh103**

### CODE REVIEW

**Comment by hungnh103** [comment]
ready

**Comment by hungnh103** [comment]
closed

**Comment by hungnh103** [comment]
ready

**Comment by hungnh103** [comment]
merged

File: app/services/hook_service.rb [hunk 1]
```diff
@@ -16,7 +16,8 @@ def initialize payload
   def valid?
     return false unless MESSAGES_VALID.include?(@comment_body)
 
-    if @sender == @owner || WhiteList.first.include?(@sender["login"])
+    if (@sender == @owner && (@comment_body == "ready" || @comment_body == "closed")) ||
```
Comment (dieunb) [inline app/services/hook_service.rb]: `(["ready", "closed"].include?(@comment_body))`

**Comment by hungnh103** [comment]
closed

**Comment by hungnh103** [comment]
closed

**Comment by hungnh103** [comment]
ready

**Comment by hungnh103** [comment]
closed

**Comment by hungnh103** [comment]
closed

**Comment by hungnh103** [comment]
closed

**Comment by hungnh103** [comment]
ready

**Comment by hungnh103** [comment]
merged

**Comment by hungnh103** [comment]
ready

**Comment by hungnh103** [comment]
merged

**Comment by hungnh103** [comment]
merged

**Pull request merged**